src/test/java/uk/gov/hmcts/reform/cosapi/handler/GlobalExceptionHandlerTest.java

```diff
@@ -0,0 +1,142 @@
+package uk.gov.hmcts.reform.cosapi.handler;
+
+import feign.FeignException;
+import feign.Request;
+import feign.RequestTemplate;
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.cosapi.exception.CaseCreateOrUpdateException;
+import uk.gov.hmcts.reform.cosapi.exception.DocumentUploadOrDeleteException;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
```

Comment:
Why is this annotation needed?

Reply:
@ExtendWith is to register extensions
@SpringBootTest is specified on the test class that runs Spring Boot based tests
@SuppressWarnings("PMD") - required to remove errors highlighted during build